pkg/models/Models.go

```diff
@@ -19,6 +19,8 @@ type NodeDBJson struct {
 	NodeObj   Node
 	ClusterID string
 	ExpiresAT string
+	Offline   bool
```

Comment:
based off our decision here https://github.com/sogwiz/vmrenter/pull/7/files#diff-bf1eee6244c5481b1babe4ebe76f26d0R15 , this should be "isOffline"

Reply:
fixed